jsk_fetch_robot/jsk_fetch_gazebo_demo/package.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>jsk_fetch_gazebo_demo</name>
+  <version>0.0.1</version>
+  <description>Fetch gazebo demo for roseus implementation</description>
+
+  <maintainer email="yamaguchi@jsk.imi.i.u-tokyo.ac.jp">Naoya Yamaguchi</maintainer>
+
+  <license>BSD</license>
+
+  <buildtool_depend>catkin</buildtool_depend>
+
+  <exec_depend>fetch_gazebo_demo</exec_depend>
+  <exec_depend>fetcheus</exec_depend>
+
+  <!-- to avoid roslaunch-check, this should be include in fetch_gazebo -->
+  <exec_depend>robot_state_publisher</exec_depend>
```

Comment:
https://github.com/fetchrobotics/fetch_gazebo/pull/108
this problems is solved in melodic, but can you add this URL?

Reply:
It seems that the PR is not included in the newest `fetch_gazebo`
https://github.com/fetchrobotics/fetch_gazebo

I add the following commit.
https://github.com/jsk-ros-pkg/jsk_robot/pull/1213/commits/4c51d4ce0c0c3c7469680763f3edd7f64085c793